tests/frontends/torch/test_torch_activation.py

```diff
@@ -38,5 +38,80 @@ def test_hardswish(shape, dtype):
     check_module(torch.nn.Hardswish(), [torch.randn(shape, dtype=dtype)])
 
 
```

Comment:
Seems miss the tests for conv1d, conv1d_transpose, ...

Reply:
Hey @yaoyaoding, currently working on those tests